libparsec/crates/platform_storage/Cargo.toml

```diff
@@ -39,6 +41,8 @@ serde-wasm-bindgen = { workspace = true }
 libparsec_tests_lite = { workspace = true }
 # Note `libparsec_tests_fixtures` enables our `test-with-testbed` feature
 libparsec_tests_fixtures = { workspace = true, default-features = false, features = ["test-with-platform-storage-testbed"] }
+libparsec_platform_storage = {path = ".",features = ["expose-test-methods"]}
 
 [target.'cfg(target_arch = "wasm32")'.dev-dependencies]
 wasm-bindgen-test = { workspace = true }
+libparsec_platform_storage = {path = ".",features = ["expose-test-methods"]}
```

Comment:
The first include in `dev-dependencies` is not enough ?

Reply:
Indeed